python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/schedule_builder.py

```diff
@@ -0,0 +1,158 @@
+from collections.abc import Mapping, Sequence
+from typing import Any, Optional
+
+from dagster import AssetsDefinition, ScheduleDefinition, get_dagster_logger
+from dbt.cli import dbt_cli
+from dbt.cli.options import MultiOption
+
+from dagster_dbt.asset_utils import (
+    DAGSTER_DBT_EXCLUDE_METADATA_KEY,
+    DAGSTER_DBT_INDIRECT_SELECTION_METADATA_KEY,
+    DAGSTER_DBT_OTHER_ARGS_METADATA_KEY,
+    DAGSTER_DBT_SELECT_METADATA_KEY,
+    build_schedule_from_dbt_selection,
+)
+from dagster_dbt.cloud_v2.resources import (
+    DBT_CLOUD_DEFAULT_EXCLUDE,
+    DBT_CLOUD_DEFAULT_SELECT,
+    DbtCloudWorkspace,
+)
+from dagster_dbt.cloud_v2.types import DbtCloudJob
+
+logger = get_dagster_logger()
+
+
+SELECT_ARGUMENT_KEY = "select"
+EXCLUDE_ARGUMENT_KEY = "exclude"
+INDIRECT_SELECTION_KEY = "indirect_selection"
+
+
+def execute_step_to_command_args(execute_step: str) -> Sequence[str]:
+    execute_step_parts = execute_step.split()
+    if execute_step_parts[0] == "dbt":
+        execute_step_parts.pop(0)
+    return execute_step_parts
+
+
+def multi_option_arg_value_to_str(arg_value: Sequence[tuple[str]]) -> str:
+    return " ".join(arg_value[0])
+
+
+def reconstruct_args_str(
```

Comment:
In dbt Cloud, an execute step is a standard dbt command. After parsing the selection args, we must reconstruct the args string, which will be appended to the args in DbtCloudWorkspace.cli(...).